source/_integrations/tesla.markdown

```diff
@@ -35,13 +35,14 @@ Home Assistant offers the Tesla integration through **Configuration** -> **Integ
 
 Enter username and password and then continue.
 
-Alternatively, Home Assistant will also load Tesla via the  `configuration.yaml`. Add the following to your `configuration.yaml` file:
+Alternatively, Home Assistant will also load Tesla via the  `configuration.yaml` and update the integration information. Add the following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 tesla:
   username: YOUR_EMAIL_ADDRESS
   password: YOUR_PASSWORD
+  verify_ssl: True
```

Comment:
This changes the YAML configuration for this integration that communicates with a service.
As per ADR-0010, we no longer accept changes to the YAML configuration for these integrations.

For more information see: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>